vertx-mssql-client/src/main/java/io/vertx/mssqlclient/impl/codec/CursorData.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2011-2022 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.mssqlclient.impl.codec;
+
+/**
+ * Data related to a server cursor.
+ */
+public class CursorData {
```

Comment:
consider making it package private if the class is not shared outside

Reply:
Done